### ISSUE TITLE
Shutdown WOL when computer dont have password?

### ISSUE THREAD

**Issue by viponja** [basildane/WakeOnLAN#95]
I try to shutdown computer who dont have a password. Only user and computer name. I have domain, 
but this computer is not turned on. Is it possible to shut down a computer that, it is not the only one who not join in domain and does not have a password on the computer? sry for my english

I love your program WOL, and this program working great for me in domain.

**Comment by basildane** [comment]
Wake Up does not use any password.
Shutdown requires a username and password.  If the machine is in a WorkGroup, then use the machine name as the "domain".

The username / password must have the rights to shutdown the remote computer.  For example, an administrator.

**Comment by viponja** [comment]
Yes I know that, I try to shutdown in WorkGroup, but when I using computer name as a domain, and username Home this is administrator, I dont have password, then when I try to shutdown throw me a mistake Error: Access i denied. (Exception from HRESULT:0x80070005 (E_ACCESSDINEID))
WakeUp working fine, but he can put a computer out if he is not in the domain, and others are?

**Comment by contain02** [comment]
I also encountered this problem,I would like to ask how to solve it.

**Comment by contain02** [comment]
Policy group Disable blank password only allows consistence logins to solve this problem.
